src/pathfora.js

```diff
@@ -694,6 +705,40 @@
     },
 
     /**
+     * @description check if a customTrigger widget
+     * is ready to be displayed, and if so display the widget
+     * @param {object} widget
+     */
+    triggerWidget: function (widget) {
+      var valid;
+
+      for (var key in widget.watchers) {
```

Comment:
cool, lets just make a ticket and address it down the road. not going to effect n e thing in the interim or prevent backwards compatibility at all.